packages/visual-dev/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import styled from 'styled-components'
+
+import * as Styles from './Styles'
+
+interface AlertProps {
+	type: "critical" | "warning" | "success" | "info"
+	title: string
+	children: React.ReactNode
+}
+
+const AlertDiv = styled.div<Pick<AlertProps, 'type'>>`
+	${Styles.base}
+	${props => Styles[props.type]}
+`
+
+export const Alert: React.FC<AlertProps> = ({
+	type,   
+	title,
+	children
+}) => {
+	return (
+		<AlertDiv type={type}>
+			{SVG[type]}
+			<div style={{paddingLeft: 16}}>
+				<div>
```

Comment:
Maybe just remove this extra div? Doesn't seem like it does anything since there's another div around it

Reply:
done!